easybuild/tools/options.py

```diff
@@ -1166,6 +1167,11 @@ def show_system_info(self):
             "  -> vendor: %s" % system_info['cpu_vendor'],
             "  -> architecture: %s" % get_cpu_architecture(),
             "  -> family: %s" % get_cpu_family(),
+        ]
+        if cpu_arch_name:
+            lines.append("  -> arch name: %s" % cpu_arch_name)
```

Comment:
I think there is a small inconsistency... if here you mean exactly this, then without archspec it will still be printed, but UNKNOWN, and in `test_show_system_info` there is no reason to check for HAVE_ARCHSPEC, since the pattern will always be there

otherwise this should be `if cpu_arch_name != UNKNOWN`, no?

Reply:
Maybe we should hint that `archspec` is not installed when `cpu_arch__name == UNKNOWN`?